api/src/main/resources/metadata/Custom_Concepts.xml

```diff
@@ -5265,6 +5346,92 @@
+    <concept_name  concept_name_id="161530"  concept_id="166193"  name="Child alive SB/NND"  locale="en"  locale_preferred="1"  creator="1"  date_created="2021-02-02 16:44:35"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="95a34b22-c107-4a45-8197-94de2f29463a"  />
+    <concept_name  concept_name_id="161531"  concept_id="166194"  name="Immunized"  locale="en"  locale_preferred="1"  creator="1"  date_created="2021-02-02 16:45:33"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="4005e520-2311-4950-b57a-7f608f864eed"  />
+    <concept_name  concept_name_id="161532"  concept_id="166195"  name="health condition"  locale="en"  locale_preferred="1"  creator="1"  date_created="2021-02-02 16:46:29"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="23daf136-4042-4bcb-8336-261c17210b79"  />
+    <concept_name  concept_name_id="161533"  concept_id="166196"  name="WOA"  locale="en"  locale_preferred="1"  creator="1"  date_created="2021-02-02 16:50:46"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="81fd10c7-1e9e-4adb-ab75-5049ec0496b1"  />
+    <concept_name  concept_name_id="161534"  concept_id="166197"  name="Fundal height"  locale="en"  locale_preferred="1"  creator="1"  date_created="2021-02-02 16:52:12"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="464482d2-3a03-4e3f-92cf-608ed4a42649"  />
+    <concept_name  concept_name_id="161535"  concept_id="166198"  name="Presentation"  locale="en"  locale_preferred="1"  creator="1"  date_created="2021-02-04 09:26:04"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="8c201f6b-5393-436e-a106-a820228b3841"  />
+    <concept_name  concept_name_id="161536"  concept_id="166199"  name="Position/lie"  locale="en"  locale_preferred="1"  creator="1"  date_created="2021-02-04 09:28:18"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="21327db6-0c7e-43f3-95ac-e4559706cf7f"  />
+    <concept_name  concept_name_id="161537"  concept_id="166200"  name="Relation PP/Brim"  locale="en"  locale_preferred="1"  creator="1"  date_created="2021-02-04 09:30:02"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="aa88f4fd-cf47-4a82-bf7a-ec424602a81a"  />
+    <concept_name  concept_name_id="161538"  concept_id="166201"  name="Foetal heart rate"  locale="en"  locale_preferred="1"  creator="1"  date_created="2021-02-04 09:31:46"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="27c287ce-f2a4-442b-bf25-411c0b426ce6"  />
+    <concept_name  concept_name_id="161539"  concept_id="166202"  name="pH"  locale="en"  locale_preferred="0"  creator="1"  date_created="2021-02-04 11:05:36"  concept_name_type="NULL"  voided="1"  voided_by="1"  date_voided="2021-02-04 11:13:52"  void_reason="Deleted due to change of name"  uuid="a26fc661-0896-4ad9-b45a-141f174ec167"  date_changed="2021-02-04 11:13:53"  changed_by="1"/>
```

Comment:
Why do you have a concept_name_type="NULL" you either leave it out or give it (FULLY_SPECIFIED,INDEX_TERM,SHORT).

Reply:
This is resolved